i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/connectors/README.mdx

```diff
@@ -0,0 +1,117 @@
+import ConnectorPlatformIntro from './fragments/_connector_platform_intro.mdx';
+import ConnectorStorageIntro from './fragments/_connector_storage_intro.mdx';
+import ConnectorTypeIntro from './fragments/_connector_type_intro.mdx';
+
+# 🔌 连接器
+
+## 介绍
+
+### 什么是连接器？
+
+_连接器_ 在 Logto 中扮演着重要的角色。在他们的帮助下，Logto 使得终端用户能够使用无密码注册登录和社交账号登录。在网站和应用逐渐渗透到我们的生活中的大背景下，无密码登录和社交登录让用户们能够避免为了管理各种各样的账号而感到头大。
+
+:hushed: 是不是听起来非常引人入胜？ 如果你想要更改一个已有连接器的设置，请参考「[配置连接器](../../recipes/configure-connectors/)」。如果找不到想要更改设置的连接器，你可以根据「[创建连接器](../../recipes/create-your-connector/)」的指南来开发属于你的连接器。
+
+## 组成
+
+连接器的数据中有很多的参数。
+
+为了让数据的加载和更新更加高效，我们把一部分经常更新的连接器数据托管到数据库，剩下的一部分保存在本地缓存中。
+
+<ConnectorStorageIntro />
+
+## 连接器的 _本地储存_：_ConnectorMetadata_
+
+### id
+
+_id_ 是一个用来在 Logto 中 **唯一** 标识连接器的字符串类型参数。
+
+他由连接器的开发者指定，并会被同步到数据库中。
+
+### target
+
+_target_ 是用于区别社交身份源的小写英文字符串。
+
+对于 _email_ 和 _SMS_ 连接器来说，_target_ 就好像一个人类可读的 ID。
+
+下面看一些关于 _target_ 的样例：*微信连接器网页版*和*微信连接器原生版*的 _target_ 是 `wechat`，*阿里云邮件推送连接器*的 _target_ 是 `aliyun-dm`，*阿里云短信连接器*的 _target_ 是 `aliyun-sms`。
+
+### type
+
+_type_ 的值应该是下列之一：
+
+<ConnectorTypeIntro />
+
+### platform
+
+_platform_ 的值只能是 `NULL` 或者下列的字符之一：
+
+<ConnectorPlatformIntro />
+
+:::note
+_Email 连接器_ 和 _SMS 连接器_ 的 _platform_ 总应该是 `NULL`。<br/>
+**只有** _Social 连接器_ 可以有非 `NULL` 的 _platform_ 值。
+:::
+
+### name
+
+_name_ 是一个 key 为 i18n 国家地区代码，value 是连接器用以显示的名称的对象。
+
+### logo
+
+_logo_ 是一个用以记录连接器的 logo 的相对路径或者 URL 的参数。
+
+### logoDark
+
+_logoDark_ 是一个记录连接器深色模式 logo 的相对路径或者 URL 的参数，可以为空，意味着没有为深色模式单独设计 logo。
+
+:::note
+_logo_ 总是必填的，同时 _logoDark_ 是选填的。
+
+_logo_ 在浅色模式时显示，而如果在深色模式时 _logoDark_ 存在则会显示，否则会在深色模式显示 _logo_。
+:::
+
+### description
+
+_description_ 是一个 key 为 i18n 国家地区代码，value 为连接器简介的对象。
+
+### readme
+
+_readme_ 是一个记录连接器 README 文档的相对路径。该文档内容在设置连接器时会显示。
+
+### configTemplate
+
+_configTemplate_ 是一个记录连接器配置样例文件的相对路径。
+
+## 连接器的 _远端储存_：数据库
+
+### id
+
+_id_ 是一个在数据库中可以用以 **唯一** 确定连接器的参数，并且也是映射 [_ConnectorMetadata_](./#连接器的-本地储存connectormetadata) 和连接器数据库参数的桥梁。
+
+### enabled
+
+_enabled_ 是一个默认值为 `FALSE` 的布尔变量。它是控制连接器使用状态的开关。
+
+通过把该值从 `TRUE` 置为 `FALSE` 可以把已配置好正在使用的连接器关闭。
+
+### config
+
+_config_ 可以是一个任意非空的对象。
+
+它是连接器用以储存配置的字段。每个连接器都有不同的 _config_ 参数，并且为了保证连接器能正常使用，在同步至数据库之前都会强制需要被校验（不同的连接器对于能通过校验有不同的标准）。
+
+连接器的开发者需要为他们的连接器实现一个 _config_ 的校验标准，详见「[创建连接器](../../recipes/create-your-connector/)」。
+
+想要一睹 _config_ 真容？查看「[配置连接器](../../recipes/configure-connectors/)」或者管理控制台里每个连接器的配置界面。
+
+:::note
+当前版本（v1.0）的 Logto 只支持同时有一个 _SMS/Email 连接器_ 的 _enabled_ 为 `TRUE`，也就意味着一旦一个其他的 _SMS_（或者 _Email_）类型连接器被启用时，其他所有的 _SMS_（或者 _Email_）类型连接器会被关闭（即 _enabled_ 被置为 `FALSE`）。
```

Comment:
we don't need to mention the version since the doc itself it's versioned
```suggestion
当前版本的 Logto 只支持同时有一个 _SMS/Email 连接器_ 的 _enabled_ 为 `true`，也就意味着一旦一个其他的 _SMS_（或者 _Email_）类型连接器被启用时，其他所有的 _SMS_（或者 _Email_）类型连接器会被关闭（即 _enabled_ 被置为 `false`）。
```

Reply:
btw we use "邮件连接器" and "短信连接器" in tutorials. should we align?

Reply:
aligned in this pr, let me check the following two.